opentracing/ext/tags.py

```diff
@@ -25,17 +25,27 @@
 # PeerXXX tags can be emitted by either client-side of server-side to describe
 # the other side/service in a peer-to-peer communications, like an RPC call.
 
-# PeerService records the service name of the peer
+# PEER_SERVICE records the service name of the peer
 PEER_SERVICE = 'peer.service'
 
-# PeerHostname records the host name of the peer
+# PEER_HOSTNAME records the host name of the peer
 PEER_HOSTNAME = 'peer.hostname'
 
-# PeerHostIPv4 records IP v4 host address of the peer
+# PEER_HOST_IPV4 records IP v4 host address of the peer
 PEER_HOST_IPV4 = 'peer.ipv4'
 
-# PeerHostIPv6 records IP v6 host address of the peer
+# PEER_HOST_IPV6 records IP v6 host address of the peer
 PEER_HOST_IPV6 = 'peer.ipv6'
 
-# PeerPort records port number of the peer
+# PEER_PORT records port number of the peer
 PEER_PORT = 'peer.port'
+
+# SPAN_KIND hints at relationship between spans, like parent/child
+SPAN_KIND = 'span.kind'
+
+# Marks a span representing the client-side of an RPC or other remote call
+SPAN_KIND_RPC_CLIENT = 1
```

Comment:
oh, good to know. good point that impls can decide how to encode/decode these (a nice property) so really the value is not meaningful. I now care less about this if you would rather use ints :)